packages/java/parser-jvm-plugin-nonnull/src/test/java/com/vaadin/hilla/parser/plugins/nonnull/nullable/NullableEndpoint.java

```diff
@@ -0,0 +1,20 @@
+package com.vaadin.hilla.parser.plugins.nonnull.nullable;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Version;
+
+@Endpoint
+public class NullableEndpoint {
+
+    public NullableFieldModel nullableFieldModel(
+            NullableFieldModel nullableFieldModel) {
+        return nullableFieldModel;
+    }
+
+    public static class NullableFieldModel {
+        @Id
```

Comment:
Those fields look nullable by default, unless I'm missing something. You should probably test them under a `@NonNullApi` annotation.

Reply:
I should, but I should also check that they are indeed nullable without any other annotation. 
I will add another test.